tools/shellcheck.sh

```diff
@@ -19,4 +19,4 @@ if ! [ -x "$(command -v shellcheck)" ]; then
 fi
 
 # TODO - fix warnings in .buildkite/run-amd-test.sh
-find . -name "*.sh" -not -path "./.buildkite/run-amd-test.sh" -print0 | xargs -0 -I {} sh -c 'git check-ignore -q "{}" || shellcheck "{}"'
```

Comment:
this is a bug that, when I change some sh file, `./.git/logs/refs/remotes/ykc/format.sh` will be checked.